oca_dependencies.txt

```diff
@@ -1 +1,2 @@
 # See https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#oca_dependencies-txt
+server-ux
```

Comment:
don't need this (see other comment in https://github.com/OCA/partner-contact/pull/1108)